mat64/hogsvd.go

```diff
@@ -111,6 +118,29 @@ func (gsvd *HOGSVD) Factorize(m ...Matrix) (ok bool) {
 	return true
 }
 
+// TODO(kortschak): Export this as SolveTwoChol.
+// solveTwoChol finds the matrix m that solves A * m = B where A and B are represented
+// by their Cholesky decompositions a and b, placing the result in the receiver.
+func (m *Dense) solveTwoChol(a, b *Cholesky) error {
+	if !a.valid() || !b.valid() {
+		panic(badCholesky)
+	}
+	an := a.chol.mat.N
+	bn := b.chol.mat.N
+	if an != bn {
+		panic(matrix.ErrShape)
+	}
+
+	m.reuseAsZeroed(bn, bn)
+	m.Copy(b.chol)
+	blas64.Trsm(blas.Left, blas.Trans, 1, a.chol.mat, m.mat)
```

Comment:
Me either. Here is what I think it should be

````
Σ_A ^-1 Σ_B

(A^T * A)^-1 * (B^T * B)
A^-1 * A^-T * B^T * B
A^-1 * C * B
D * B
````
To get C, we have `A^T* C = B^T`, which is Trsm with Trans, Left, but also B.T() instead of just B. You have the trans and left, but it seems like the wrong RHS.
Then we have `A * D = C` which is the Left and NoTrans that you have.
It seems strange that this works despite totally dropping the last B. You seem to be computing `A^-1 * A^-T * B`, which doesn't seem like the same thing.